persist_error/tests/test_handler.py

```diff
@@ -113,6 +114,8 @@ def test_subsequent_retry_increment(self, mock_sm):
     @mock.patch('persist_error.handler.send_message', autospec=True)
     @mock.patch('persist_error.handler.send_notification', autospec=True)
     def test_terminal_failure_behavior(self, mock_sn, mock_sm):
+        os.environ['AWS_TERMINAL_QUEUE_URL'] = 'terminal_queue'
```

Comment:
I would prefer this done with @mock.patch.dict to maintain consistency with the other tests in this module.

Reply:
ok